src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -35,10 +35,17 @@ object Lorre extends App with LazyLogging {
   lazy val db = DatabaseUtil.db
   val tezosNodeOperator = new TezosNodeOperator(TezosNodeInterface)
 
+  val numberOfRetries = conf.getInt("lorre.numberOfRetries")
+  @tailrec
+  def retry[A](attempts: Int)(fn: => Try[A]): Try[A] = fn match {
```

Comment:
This looks like a general purpose function which may be useful somewhere else too.
Would it make sense to put it in a separate file in the `util` package?

Reply:
Agreed with @ivanopagano .

Reply:
Sounds good.